dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -224,6 +224,32 @@ object WavesDexCli extends ScoptImplicits {
     }
   }
 
+  def checkConfig(args: Args): Unit = {
+    def prettyPrintUnusedProperties(unusedProperties: Seq[String]): Unit = {
+      if (unusedProperties.nonEmpty) {
+        println(s"Warning! Found ${unusedProperties.size} potentially unused properties.")
+        println("Unused matcher properties found:")
```

Comment:
As I see in tests, we crop "waves.dex.". How about to make things a little clearer, e.g.: "Unused matcher properties found in waves.dex:" ?